sklearn/feature_extraction/tests/test_dict_vectorizer.py

```diff
@@ -110,3 +110,13 @@ def test_deterministic_vocabulary():
     v_2 = DictVectorizer().fit([d_shuffled])
 
     assert v_1.vocabulary_ == v_2.vocabulary_
+
+
+def test_n_features_in():
+    # For vectorizers, n_features_in_ does not make sense and it is always
+    # None
```

Comment:
Actually, it not `None` but it is rather not existing.

Reply:
this is outdated indeed, thanks